package.json

```diff
@@ -208,6 +208,7 @@
     "jsdom": "^15.2.1",
     "json2csv": "^4.2.1",
     "jsonschema": "^1.1.1",
+    "just-diff": "^3.1.1",
```

Comment:
This is a library to diff two objects and return a list of differences. It's in dev dependencies. I didn't see anything similar in the existing dependencies.